.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
 
 # running tests
 script:
+  - cd /tmp
   - pytest -v --maxfail=2 --cov=jupyterhub jupyterhub/tests
```

Comment:
@minrk I am able to get the tests to run locally and on Travis by removing `cd /tmp` and adding:
```
script:
    pytest -v --basetemp=tmp --maxfail=2 --cov=jupyterhub jupyterhub/tests
```
[ref](https://docs.pytest.org/en/latest/tmpdir.html#the-default-base-temporary-directory)

Though when making this change, one test fails `test_static_files` in `test_pages.py`.  [test output link](https://travis-ci.org/willingc/jupyterhub/jobs/261573175)